800.renames-and-merges/l.yaml

```diff
@@ -68,6 +68,8 @@
 - { setname: lexicon,                  name: ["python:dns-lexicon", dns-lexicon] }
 - { setname: lgogdownloader,           name: lgogdownloader-qt5, addflavor: true }
 - { setname: liberal-crime-squad,      name: [lcs,liberalcrimesquad] }
+- { setname: libgedit-amtk,            namepat: "(?:lib)?amtk[0-9.-]*" }
```

Comment:
Better not to use regexps where it's not necessary. Just `name: [amtk, libamtk]` is sufficient here.